src/pages/DACDatasets.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState, useCallback } from 'react';
+import { button, div, h, span } from 'react-hyperscript-helpers';
+import {DataSet, User} from '../libs/ajax';
+import {Styles} from '../libs/theme';
+import {img} from 'react-hyperscript-helpers';
+import lockIcon from '../images/lock-icon.png';
+import SearchBar from '../components/SearchBar';
+import {DACDatasetTable, DACDatasetTableColumnOptions} from '../components/dac_dataset_table/DACDatasetTable';
+import {getSearchFilterFunctions, Notifications, searchOnFilteredList} from '../libs/utils';
+import {useRef} from 'react';
+import { consoleTypes } from '../components/dac_dataset_table/DACDatasetTableCellData';
+import '../components/dac_dataset_table/dac_dataset_table.css';
+
+export default function DACDatasets(props) {
+  const [datasets, setDatasets] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const searchRef = useRef('');
+  const filterFn = getSearchFilterFunctions().datasets;
+  const { history } = props;
+
+  const handleSearchChange = useCallback((searchTerms) => searchOnFilteredList(
+    searchTerms,
+    datasets,
+    filterFn,
+    setFilteredList
+  ), [datasets, filterFn]);
+
+  const getDatasets = useCallback(async () => {
+    const userDatasets = await User.getUserRelevantDatasets();
+    userDatasets.map(async dataset => {
+      const datasetObject = await DataSet.getDataSetsByDatasetId(dataset.dataSetId);
+      dataset.dacApproval = datasetObject.dacApproval;
+    });
+    return userDatasets;
```

Comment:
Ah, ok. I see the issue. `/me/dac/datasets` uses DatasetDTO objects (which are deprecated and don't include all fields. i think we should update this endpoint if we're going to still need it to return regular Dataset objects, then you shouldn't have to do this map operation